web_company_color/models/assetsbundle.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Alexandre Díaz <dev@redneboa.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.http import request
+
+from odoo.addons.base.models.assetsbundle import AssetsBundle, ScssStylesheetAsset
+
+
+class AssetsBundleCompanyColor(AssetsBundle):
+    def get_company_color_asset_node(self):
+        """
+        Process the user active company scss and returns the node to inject
+        """
+        # For some reason now need get the active company from cookies,
+        # i don't know if exists other better way to do that
+        active_company_id = int(
```

Comment:
If there is only one company available this fails as @LoisRForgeFlow  mentioned above. This does the trick:
```python
        try:
            active_company_id = int(
                request.httprequest.cookies.get("cids", "").split(",")[0]
            )
        except Exception:
            active_company_id = False
```

Reply:
Thanks!